pkg/updater/updater_test.go

```diff
@@ -229,6 +229,7 @@ func TestUpdate(t *testing.T) {
 			err := Update(
 				ctx,
 				client,
+				Metrics{},
```

Comment:
Can we unit test this new scenario? AKA successes should increment the success counter, errors should increment the error counter.